Dockerfile

```diff
@@ -151,6 +152,15 @@ COPY --from=builder \
 COPY --from=builder /build/vcftools/built/bin/    /usr/local/bin/
 COPY --from=builder /build/vcftools/built/share/  /usr/local/share/
 
+# Add Pangolin and PangoLEARN + deps
+RUN curl -fsSL https://github.com/virus-evolution/gofasta/releases/download/v0.0.6/gofasta-linux-amd64 \
+        -o /usr/local/bin/gofasta \
+  && chmod a+rx /usr/local/bin/gofasta
+RUN cd /usr/local/bin && curl -fsSL https://github.com/lh3/minimap2/releases/download/v2.24/minimap2-2.24_x64-linux.tar.bz2 \
+  | tar xjvpf - --strip-components=1 minimap2-2.24_x64-linux/minimap2
+RUN pip install git+https://github.com/cov-lineages/pangolin.git@v2.3.8
+RUN pip install git+https://github.com/cov-lineages/pangoLEARN.git@2021-04-01
```

Comment:
The pango versions mentioned in the original issue that this PR addresses are pretty old now. I think we'll want the latest versions of `pangolin` ([3.1.17](https://github.com/cov-lineages/pangolin/releases/tag/v3.1.17)) and pangoLEARN ([2021-12-06](https://github.com/cov-lineages/pangoLEARN/releases/tag/2021-12-06)).

It looks like [pangolin version 3.0+ requires UShER](https://cov-lineages.org/resources/pangolin/installation.html), too. UShER provides [an installation script for Ubuntu](https://github.com/yatisht/usher/blob/master/install/installUbuntu.sh) that we could use as a starting point.

Reply:
Ok - I've managed to get Pangolin 3.x with UShER functionality working in the image build, but if I'm reading the Pango docs correctly, UShER is an *optional* dependency for pangolin ([docs](https://cov-lineages.org/resources/pangolin/usage.html)) and I don't see anything in nexstrain that enables that functionality, so I'm omitting it.